LoginRegister.DBO/DBO/LoginRegister.cs

```
@@ -0,0 +1,256 @@
+﻿using LoginRegister.DBO.Models;
+using LoginRegisterAPI.Helper;
+using LoginRegisterAPI.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace LoginRegister.DBO.DBO
+{
+    public class LoginRegister
+    {
+        private readonly HospitalManagementContext _context;
+
+        public LoginRegister(HospitalManagementContext ctx)
+        {
+            _context = ctx;
+        }
+
+        //public LoginRegister()
+        //{
+        //}
+
+        public Login VerifyLogin(string email, string password)
+        {
+            Login user = new Login();
+            try
+            {
+
+                //user = _context.Login.Where(e => e.UserId == username && e.Password == password).Select(e => new Login
+                //{
+                //    UserId = e.UserId,
+                //    DefaultRole = e.DefaultRole
+
+                //}).FirstOrDefault();
+
+                user = _context.Login.Where(e => e.Email == email && e.Password == password).Select(e => new Login
+                {
+                    Email = e.Email,
+                    DefaultRole = e.DefaultRole
+
+                }).FirstOrDefault();
+
+            }
+            catch (Exception ex)
+            {
+
+            }
+            return user;
+        }
+
+        public bool CheckUserExist(string userId)
+        {
+            bool success = false;
+            try
+            {
+                var user= _context.Login.Where(e => e.UserId == userId).SingleOrDefault();
+                if (user != null)
+                  return  success = true;
+            }
+            catch(Exception ex)
```

empty catch block is of no use.. Either log exception, throw it or do not add try... catch